tests/driver.c

```diff
@@ -211,7 +228,9 @@ int main(void) {
   RUN_TEST(test_generate_address);
   RUN_TEST(test_get_tips_pair);
   RUN_TEST(test_get_tips);
+  // send transaction before receiving transaction
```

Comment:
What is the comment referring to?

Reply:
This is reminder in case the ordering being changed causing driving test crash

Reply:
Don't put such notice inside `main()`. Instead, you can write down internals of test suite or improve each test case.

Reply:
Since this file was intended to behave as test driver, it should be decoupled from exact test logic.

Reply:
I will separate them into different test suite